virtcontainers/qemu_amd64.go

```diff
@@ -171,7 +184,7 @@ func (q *qemuAmd64) appendIOMMU(devices []govmmQemu.Device) []govmmQemu.Device {
 	iommu := govmmQemu.IommuDev{
 		Intremap:    true,
 		DeviceIotlb: true,
-		Caching:     true,
+		CachingMode: true,
```

Comment:
What are the use cases for `caching-mode=off`? Do we want to warn in that case?

Reply:
AFAIK caching-mode=on is only needed for device passthrough. So a non-configurable "true" value in kata makes sense.
I don't know if we should log at the govmm level though, it is supposed to be a generic library

Reply:
Yes with `intel_iommu`, `caching-mode=on` is must otherwise device assignment will fail